packages/peregrine/lib/talons/Navigation/__tests__/useNavigation.spec.js

```diff
@@ -58,9 +58,9 @@ jest.mock('@apollo/client', () => {
     const apolloClient = jest.requireActual('@apollo/client');
     return {
         ...apolloClient,
-        useQuery: jest
-            .fn()
-            .mockReturnValue({ data: { storeConfig: { root_category_id: 1 } } })
+        useQuery: jest.fn().mockReturnValue({
+            data: { storeConfig: { root_category_uid: 'Mg==' } }
```

Comment:
@itsyanik, here you also need to change the `useCatalogContext` mock on line 27 to use `uid` values as keys, same thing on the `setCategoryId` calls, they need to pass `uid` strings instead of numbers.